crates/swc_ecma_codegen/Cargo.toml

```diff
@@ -14,6 +14,7 @@ bitflags = "1"
 memchr = "2.4.1"
 num-bigint = {version = "0.2", features = ["serde"]}
 once_cell = "1.9.0"
+regex = "1"
```

Comment:
Can you remove this line?

Reply:
Also, please revert `Cargo.lock`

Reply:
resolved in 0ef810b4a4d699c0ebf59a02d5cc139e0f41c062